src/frontend/services/NotesServices/updateNoteService.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+const updateNoteService = async (token, note) => {
+  try {
+    await axios.post(
+      `/api/notes/${note._id}`,
+      { note },
+      {
+        headers: { authorization: token },
+      }
+    );
+  } catch (e) {
+    console.log(e);
```

Comment:
Please remove logs.

Reply:
I will display the error message in add toast in coming days